packages/backend/src/core/activity/counters/StarkexCounter.ts

```diff
@@ -63,6 +63,5 @@ export function createStarkexCounter(
 
 /** @internal */
 export function getStarkexLastDay(timestamp: UnixTime) {
-  // StarkEx APIs have inherent delay
-  return timestamp.add(-1, 'days').add(-4, 'hours').toStartOf('day').toDays()
+  return timestamp.toStartOf('day').toDays()
```

Comment:
Isn't this gonna show 0 tx for the past day tps all starkex projects? Also, we'll have our frontend stop building because of the sanity check.

Reply:
That's why we have `uncertaintyBuffer`for, right? We can record data as fast as possible and we will simply update what's in the database.

Reply:
Yes, it will sync eventually, but the sanity check will probably be never passed